app.js

```diff
@@ -1,9 +1,32 @@
 const express = require("express");
+const swaggerUi = require("swagger-ui-express");
+const swaggerSpec = require("./swagger.js");
 const app = express();
 const port = 3000;
 
+app.use("/health-check", swaggerUi.serve, swaggerUi.setup(swaggerSpec));
+/**
+ * @swagger
+ * /health-check:
+ *   get:
+ *     summary: Check the health of the application
+ *     description: |
+ *       Returns a message confirming the health of the application and the port it's running on.
+ *     responses:
+ *       200:
+ *         description: Health check response
+ *         content:
+ *           text/plain:
+ *             schema:
+ *               type: string
+ */
+
 app.get("/health-check", (req, res) => {
-	res.send(`App server is running on port ${port}`);
+	res.send(`Health check on port ${port}`);
```

Comment:
You have the same route as on the line 7, it will be not working

Reply:
I have added extra route for swagger documents